spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -67,4 +73,32 @@ public Subscriber getSubscriber(String subscriptionName, MessageReceiver receive
 				.setCredentialsProvider(this.credentialsProvider)
 				.build();
 	}
+
+	@Override
+	public PullRequest getPullRequest(String subscriptionName, Integer maxMessages,
```

Comment:
That's not `get` buy its logic. This is really `create` or `make`.
The `get` assumes to be idempotent and should return the same object for each call.

Reply:
In that case the `getSubscriber` above should also be renamed.